corehq/apps/registration/static/registration/js/new_user.ko.js

```diff
@@ -37,34 +36,6 @@ module.onModuleLoad = function () {
     throw new Error("overwrite onModule load to remove loading indicators");
 };
 
-// Can't set up analytics until the values for the A/B tests are ready
-kissmetrics.whenReadyAlways(function () {
-    kissmetrics.track.event("Viewed CommCare signup page");
-
-    module.submitSuccessAnalytics = function (data) {
-        kissmetrics.track.event("Account Creation was Successful");
-
-        var appcuesEvent = "Assigned user to Appcues test",
-            appcuesData = {
-                'Appcues test': data.appcuesAbTest,
-            };
-
-        appcues.identify(data.email, appcuesData);
-        appcues.trackEvent(appcuesEvent, appcuesData);
-
-        kissmetrics.identify(data.email);
-        kissmetrics.identifyTraits(appcuesData);
-        kissmetrics.track.event(appcuesEvent, appcuesData);
-
-        if (data.deniedEmail) {
-            kissmetrics.track.event("Created account after previous denial due to enterprise restricting signups", {
-                email: data.email,
-                previousAttempt: data.deniedEmail,
-            });
-        }
-    };
-});
```

Comment:
Thought I left this comment in my review but I guess I didn't. Assuming removing kissmetrics usage was also fine because we are ultimately removing that as well, but just calling it out since it wasn't explicitly mentioned in the PR.

Reply:
Ah yes, I should have called that out. There's some app cues calls in here, but rather than surgically removing it in this PR, I just removed the whole block, because as you said I knew I was about to remove Kissmetrics as well in the next PR, and it didn't seem worth the fuss.